Sources/SwiftGRPCNIO/GRPCChannelHandler.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import SwiftProtobuf
+import NIO
+import NIOHTTP1
+
+/// Processes individual gRPC messages and stream-close events on a HTTP2 channel.
+public protocol GRPCCallHandler: ChannelHandler {
+  func makeGRPCServerCodec() -> ChannelHandler
```

Comment:
Can we document all the public functions/getters for these protocols?

Reply:
Not sure what you mean with that, but I've added headerdocs for this protocol's methods now.